easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "PackedBinary"
+
+name = "I-TASSER"
+version = '4.0'
+
+homepage = 'http://zhanglab.ccmb.med.umich.edu/I-TASSER/'
+description = """I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function
+modelling and comparison."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Can't download from the web site automatically as registration is required.
+# The source code may be downloaded manually from http://zhanglab.ccmb.med.umich.edu.
+sources = ['%(name)s%(version)s.tar.bz2']
+
+dependencies = [('BLAST', '2.2.26', "-Linux_x86_64"), ('Java', '1.7.0_80')] 
+
+postinstallcmds = ['chmod a+x %(installdir)s/submit%(name)s']
```

Comment:
@valtandor where is this `submit*` script supposed to come from? it's not there in my tarballs...

```
0f5fd94168e54c25e0aa3470c84ee7c0  I-TASSER4.0.tar.bz2
ace1fcb4a7b45bf412c292d234a45658  I-TASSER4.2.tar.bz2
```